docs/en/multi_modal/llava_qwen.md

```diff
@@ -0,0 +1,172 @@
+# Llava-Qwen2
+
+LMDeploy supports the following llava-qwen2 series of models, which are detailed in the table below:
+
+|            Model            | Size | Supported Inference Engine |
+| :-------------------------: | :--: | :------------------------: |
+| Llava-interleave-qwen-7b-hf |  7B  |         TurboMind          |
+
+The next chapter demonstrates how to deploy an LlavaQwen2 model using LMDeploy, with [LlavaQwen2](https://huggingface.co/llava-hf/llava-interleave-qwen-7b-hf) as an example.
+
+## Installation
+
+Please install LMDeploy by following the [installation guide](../get_started/installation.md).
+
+Or, you can build a docker image to set up the inference environment. If the CUDA version on your host machine is `>=12.4`, you can run:
+
+```
+git clone https://github.com/InternLM/lmdeploy.git
+cd lmdeploy
+docker build --build-arg CUDA_VERSION=cu12 -t openmmlab/lmdeploy:llava_qwen2 . -f ./docker/Dockerfile
+```
+
+Otherwise, you can go with:
+
+```shell
+docker build --build-arg CUDA_VERSION=cu11 -t openmmlab/lmdeploy:llava_qwen2 . -f ./docker/Dockerfile
```

Comment:
I don't think the user need building a image, since the used Dockerfile file is same as openmmlab/lmdeploy:{tag}. The user could just use openmmlab/lmdeploy:latest-cu11/cu12